pkg/build/pipelines/git-checkout.yaml

```diff
@@ -38,6 +38,10 @@ inputs:
     description: |
       Indicates whether --recurse-submodules should be passed to git clone.
     default: false
+  
+  cherry-picks:
+    description: |
+      List of cherry picks to apply
```

Comment:
this needs information here on how it is formatted.

```
New line separated entries.  Lines can be empty. Any content on a line after `#` is ignored.
After removing comments, each line is of the form:

    [branch/]commit-id: comment explaining cherry-pick

comment and commit-id are required.  branch on origin that
the commit lives should be provided or git is not guaranteed to
have a reference to the commit-id.

Example: ...

```
cherry-picks: |
    3.10/62705d869aca4055e8a96e2ed4f9013e9917c661: